src/state.rs

```diff
@@ -418,12 +439,9 @@ mod test {
         })).is_ok());
 
         let accept_thread = thread::spawn(move || {
-            // Currently acceptor starts accepting implicitly,
-            // once fixed (https://github.com/maidsafe/crust/issues/316),
-            // the following lines should be uncommented.
-            //for a in acceptors {
-            //    assert!(::transport::accept(&a).is_ok());
-            //}
+            for a in acceptors {
+                assert!(State::accept(&a).is_ok())
+            }
```

Comment:
Yes, but now it's required for us to left these lines uncommented so we can initiate the handshake. Therefor, leaving the comment would be misleading (uncommenting code that is already uncommented).